src/tracers/print_tracer.rs

```diff
@@ -24,13 +24,21 @@ impl Tracer for PrintTracer {
 
         if matches!(opcode_variant, ZKOpcode::UMA(UMAOpcode::HeapWrite)) {
             let mut new_vm = vm.clone();
-            let (src0, src1) = address_operands_read(&mut new_vm, opcode);
+            let (src0, src1) = address_operands_read(&mut new_vm, opcode).unwrap();
             let value = src1.value;
             if value == debug_magic {
                 let fat_ptr = FatPointer::decode(src0.value);
                 if fat_ptr.offset == DEBUG_SLOT {
-                    let how_to_print_value = new_vm.current_frame_mut().heap.read(DEBUG_SLOT + 32);
-                    let value_to_print = new_vm.current_frame_mut().heap.read(DEBUG_SLOT + 64);
+                    let how_to_print_value = new_vm
+                        .current_frame_mut()
+                        .unwrap()
+                        .heap
+                        .read(DEBUG_SLOT + 32);
+                    let value_to_print = new_vm
+                        .current_frame_mut()
+                        .unwrap()
```

Comment:
Ditto

Reply:
Removed all `unwrap()`s related to `EraVmError` and only left the ones related to `U256::from_str_radix()` that we know won't fail.